### PR TITLE
Copyedited Documentation

### LINKED ISSUE #1401: Copyediting of documentation
## Enhancement proposal
As I look through the docs, I'll make a list here of pages that could do with some copyediting. Most of these pages are relatively well-written and only need some minor corrections. I think it'll be a good task for new contributors who are proficient with English.

- [ ] `installation/win`
- [ ] `installation/troubleshooting`
- [ ] `installation/for_dev` Last section seems incomplete
- [ ] `installation/plugins` Some commas can be semicolons
- [ ] `tutorials/building_blocks` "Mobjects are the basic building block**s**; probably other minor issues as well
- [ ] `tutorials/configuration` Some awkward phrasing
- [ ] `contributing/development` One `flake8` word isn't formatted correctly
- [ ] `contributing/internationalization` I copyedited this once as part of my review, but it could still do with another pass.

This is by no means an exhaustive list. Hopefully this helps new contributors with a keen eye for grammar and style find some ideas for ways to contribute.

## Additional comments
I was going to make a more exhaustive list, but I realized that almost every page in the documentation would be listed here if I were nitpicky enough.

### PR DESCRIPTION
**Pull request by Nikhil0504**
## Changelog / Overview

Added a link to Manim Community GitHub page in `for_dev.rst`.
Fixed :meth:`~.Mobject.get_start`  and added `roll` link in `building_blocks-rst`
Added language to code blocks in `configuration.rst`

## Motivation

The links are working due to which it is easier to access them.
## Explanation for Changes

These changes can make the reader more easier to understand and the new link can directly direct them to the required pages.
[Documentation Reference](https://manimce--1509.org.readthedocs.build)

## Further Comments

Closes #1401
## Checklist
- [x] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)
- [x] I have written a descriptive PR title (see top of PR template for examples)
- [x] I have written a changelog entry for the PR or deem it unnecessary
- [ ] My new functions/classes either have a docstring or are private
- [ ] My new functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [x] My new documentation builds, looks correctly formatted, and adds no additional build warnings

## Reviewer Checklist
- [ ] The PR title is descriptive enough
- [ ] The PR is labeled correctly
- [ ] The changelog entry is completed if necessary
- [ ] Newly added functions/classes either have a docstring or are private
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings

### CODE REVIEW

File: docs/source/installation/for_dev.rst [hunk 1]
```diff
@@ -54,7 +54,7 @@ Installing System Dependencies
 ******************************
 
 Please follow the instructions under :ref:`installing-manim` to install all
-dependencies (e.g. ``LaTeX``, ``ffmpeg``, etc.). Afterwards, proceed with the
+dependencies (e.g. ``LaTeX``, ``ffmpeg``, etc.). Afterward, proceed with the
```
Comment (Darylgolden) [inline docs/source/installation/for_dev.rst:57]: Pretty sure "afterwards" is correct.

File: docs/source/installation/plugins.rst [hunk 2]
```diff
@@ -132,8 +132,8 @@ Manim will look for ``__all__`` keyword defined in ``manim_plugintemplate`` and
 everything as a global variable one by one.
 
 If ``object_reference`` is a function, Manim calls the function and expects the
-function returns a list of modules or functions that needs to defined globally and
-it defined it.
+function returns a list of modules or functions that needs to be defined globally and
+in defined it.
```
Comment (Darylgolden) [inline docs/source/installation/plugins.rst:136]: I think the whole sentence is badly written and changing this single word doesn't really fix it. Can you rephrase the whole sentence?

File: docs/source/installation/win.rst [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
+There are two simple ways to download manim's dependencies, using the popular package
 managers `Scoop <https://shoop.sh>`_ and `Chocolatey <https://chocolatey.org/install>`_
 
 .. _scoop:
 
 Installing using Scoop
 **********************
 
-First you need to install Scoop, which is a command-line installer for Windows
+First,    you need to install Scoop, which is a command-line installer for Windows
```
Comment (Darylgolden) [inline docs/source/installation/win.rst:12]:
```suggestion
First, you need to install Scoop, which is a command-line installer for Windows
```

File: docs/source/tutorials/building_blocks.rst [hunk 4]
```diff
@@ -244,8 +244,8 @@ your scene by calling the :meth:`~.Scene.play` method.
 Put simply, animations are procedures that interpolate between two mobjects.
 For example, :code:`FadeIn(square)` starts with a fully transparent version of
 :code:`square` and ends with a fully opaque version, interpolating between them
-by gradually increasing the opacity.  :class:`.FadeOut` works in the opposite
-way: it interpolates from fully opaque to fully transparent.  As another
+by gradually increasing the opacity.  :class:`.FadeOut` oppositely: it
```
Comment (Darylgolden) [inline docs/source/tutorials/building_blocks.rst:247]: I don't think "oppositely" can be used here; I think the original phrasing was better.

**Review by Darylgolden — CHANGES_REQUESTED** [review]
Thanks for your PR! Just some suggestions.

File: docs/source/installation/for_dev.rst (hunk 1 above)
Comment (Nikhil0504) [reply docs/source/installation/for_dev.rst:57]: That’s what I thought but apparently this is grammatically correct 🧐

**Comment by Nikhil0504** [comment]
Finished all the changes and fixed the small errors.

File: docs/source/installation/plugins.rst [hunk 5]
```diff
@@ -132,8 +132,7 @@ Manim will look for ``__all__`` keyword defined in ``manim_plugintemplate`` and
 everything as a global variable one by one.
 
 If ``object_reference`` is a function, Manim calls the function and expects the
-function returns a list of modules or functions that needs to defined globally and
-it defined it.
+function to return a list of modules or functions that needs to be defined globally.
```
Comment (jsonvillanueva) [inline docs/source/installation/plugins.rst:135]:
```suggestion
function to return a list of modules or functions that need to be defined globally.
```

File: docs/source/installation/for_dev.rst (hunk 1 above)
Comment (jsonvillanueva) [reply docs/source/installation/for_dev.rst:57]: Both are correct, there is no difference between the two spellings. 'Afterward' is used in the U.S. and 'afterwards' is used elsewhere. Personally, I prefer `afterwards` though I'm from the USA.

**Review by jsonvillanueva — APPROVED** [review]
LGTM, approving with the minor change to plugins.rst

**Review by Darylgolden — CHANGES_REQUESTED** [review]
The scoop URL still isn't fixed?

**Review by Darylgolden — APPROVED** [review]
LGTM!

File: docs/source/installation/for_dev.rst (hunk 1 above)
Comment (naveen521kk) [reply docs/source/installation/for_dev.rst:57]: I like to have `Afterwards`.

File: docs/source/installation/for_dev.rst [hunk 6]
```diff
@@ -146,4 +146,4 @@ For Developers with pip
    python3 -m pip install .
 
 Please see :doc:`../contributing` for more details about contributing to Manim.
-Since `pip` doesn't implement editable installations from our pyproject.toml
+Since `pip` doesn't implement editable installations from our ``pyproject.toml``
```
Comment (naveen521kk) [inline docs/source/installation/for_dev.rst:149]: though this wouldn't be the case always. https://github.com/python/peps/pull/1944

File: docs/source/installation/for_dev.rst [hunk 7]
```diff
@@ -1,7 +1,7 @@
-Manim Installation For Developers
-=================================
+For Developers
+==============
 
-	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
+	This documentation is for developers who want to contribute to `ManimCommunity/manim <https://github.com/ManimCommunity/manim>`_.
```
Comment (naveen521kk) [inline docs/source/installation/for_dev.rst:4]: In rst document, like code, we should maintain a maximum line length of 80 chars. So bring the last word to next line.

File: docs/source/installation/troubleshooting.rst [hunk 8]
```diff
@@ -137,7 +137,10 @@ In this case, run
...
 
-If, however, ``--libgs=filename`` appears in the help, that means that your ``dvisvgm`` needs the Ghostscript library in order to support PostScript. Search for ``libgs.so`` (on Linux, probably in ``/usr/local/lib`` or ``/usr/lib``) or ``gsdll32.dll`` (on 32-bit Windows, probably in ``C:\windows\system32``) or ``gsdll64.dll`` (on 64-bit Windows, probably in ``c:\windows\system32`` -- yes 32) or ``libgsl.dylib`` (on Mac OS, probably in ``/usr/local/lib`` or ``/opt/local/lib``). Please look carefully, as the file might be located elsewhere, e.g. in the directory where Ghostscript is installed.
+If, however, ``--libgs=filename`` appears in the help, that means that your ``dvisvgm`` needs the Ghostscript library to support PostScript. Search for 
+``libgs.so`` (on Linux, probably in ``/usr/local/lib`` or ``/usr/lib``) or ``gsdll32.dll`` (on 32-bit Windows, probably in ``C:\windows\system32``) or 
+``gsdll64.dll`` (on 64-bit Windows, probably in ``c:\windows\system32`` -- yes 32) or ``libgsl.dylib`` (on Mac OS, probably in ``/usr/local/lib`` or 
+``/opt/local/lib``). Please look carefully, as the file might be located elsewhere, e.g. in the directory where Ghostscript is installed.
```
Comment (naveen521kk) [inline docs/source/installation/troubleshooting.rst:143]: Can you make this so that each line has 80 chars? (for readability)

File: docs/source/installation/for_dev.rst (hunk 7 above)
Comment (Darylgolden) [reply docs/source/installation/for_dev.rst:4]: (This doesn't appear to have the correct indentation so @Nikhil0504 please don't commit directly)

File: docs/source/installation/win.rst [hunk 9]
```diff
@@ -34,16 +34,16 @@ LaTeX Installation
 There are two ways of installing a LaTeX distribution that will be covered here:
 
 1. using MikTeX (approx. 2GB, takes a while to install, but provides "on-the-fly"
-package installation)
+package installation, recommended for beginners)
```
Comment (naveen521kk) [inline docs/source/installation/win.rst:37]: you say the same in the chocolatey part of MiKTeX installation.

File: docs/source/installation/for_dev.rst (hunk 7 above)
Comment (Nikhil0504) [reply docs/source/installation/for_dev.rst:4]: Yeah I'll do the changes with correct indentation.

File: docs/source/installation/win.rst [hunk 10]
```diff
@@ -92,13 +94,15 @@ FFmpeg installation
 
 LaTeX Installation
 ------------------
-You can install latex by either of the two methods below. MiKTex is very large (2 GB) while ManimLaTeX is small  (500Mb).
+You can install latex by either of the two methods below. MiKTex is very large 
+(2 GB) while ManimLaTeX is small  (500Mb).
 
-Using a custom distribution for Manim based on Texlive
-++++++++++++++++++++++++++++++++++++++++++++++++++++++
+Using a custom distribution for Manim based on Texlive (Recommended for beginners)
```
Comment (Darylgolden) [inline docs/source/installation/win.rst:100]: Can you make the R in recommended lowercase to be consistent with the styling?

**Review by Darylgolden — APPROVED** [review]
